src/test/python/deep_qa_test/layers/attention/attention_test.py

```diff
@@ -1,13 +1,92 @@
 # pylint: disable=no-self-use,invalid-name
 
-import numpy
+import numpy as np
```

Comment:
haha that's fine, i definitely understand. i should probably go through and retroactively fix the places i've used `np`, should be a quick grep away.